src/openapi/streaming/subscription.js

```diff
@@ -73,19 +73,14 @@ function subscribe() {
         ReferenceId: referenceId,
         KnownSchemas: this.parser.getSchemaNames(),
     });
+    const options = { body: data, headers: Object.assign({}, this.headers) };

```

Comment:
Let's for now use extend method from object utils to make sure we are backward compatible with IE 10.
I'll work separately on either moving up min browser support or adding babel transpilation for ES 2015.

Extend is available as part of object utils:
![image](https://user-images.githubusercontent.com/117785/70702246-1175e080-1cce-11ea-8f9b-cc3c36197cc6.png)

And this is example usage for cloning subscription args:
![image](https://user-images.githubusercontent.com/117785/70702337-336f6300-1cce-11ea-9731-197199dd288c.png)

Reply:
Good call 👍